public/backend/js/dataTable.js

```diff
@@ -0,0 +1,4 @@
+$(document).ready(function(){
```

Comment:
[rank A] will this file using for all of table list in your project? Will you put other js code in this file in future?